templates/taxi/car_detail.html

```diff
@@ -3,14 +3,25 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
+  </h1>
+  <div>
+    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger">
       Delete
     </a>
 
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
+    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary">
       Update
     </a>
-  </h1>
+    {% if car in user.cars.all %}
+      <a href="{% url "taxi:update-user-cars" pk=car.id %}" class="btn" style="background-color: orange; color: #fff;">
+        Delete this car from my cars
+      </a>
+    {% else %}
+      <a href="{% url "taxi:update-user-cars" pk=car.id %}" class="btn" style="background-color: green; color: #fff;">
```

Comment:
Consider using single quotes consistently in the `{% url %}` tag to match the style used elsewhere in the template. For example, `{% url 'taxi:update-user-cars' pk=car.id %}`.